core/templates/components/recommendations/post-chapter-recommendations.component.html

```diff
@@ -53,6 +53,29 @@
         {{ 'I18N_END_CHAPTER_PRACTICE_SESSION_TEXT' | translate }}
       </p>
     </div>
+    <div *ngIf="!practiceQuestionsAreEnabled" class="recommendation-card-wrapper">
+      <div class="recommendation-card">
+        <svg width="192" height="150" class="recommendation-card-svg">
+          <title>Revision card</title>
+          <a [attr.href]="getRevisionTabUrl()">
+            <rect width="192"
+                  height="150"
+                  [attr.fill]="'#be563c'">
+            </rect>
+            <image x="17"
+                   y="20"
+                   height="180"
+                   width="240"
+                   class="revision-icon"
+                   [attr.xlink:href]="getStaticImageUrl('/icons/review_icon_24px.svg')">
+            </image>
+          </a>
+        </svg>
+      </div>
+      <p [ngStyle]="{'font-style': 'italic'}">
```

Comment:
Prefer using a class.

Reply:
Can we use the bootstrap class `font-italic`?

Reply:
Done